src/util/db/dbconnection.cpp

```diff
@@ -249,6 +249,8 @@ bool initDatabase(QSqlDatabase database, StringCollator* pCollator) {
     }
     // v.data() returns a pointer to the handle
     sqlite3* handle = *static_cast<sqlite3**>(v.data());
+    // Ensure initialization. database.open() might not do it
+    sqlite3_initialize();
```

Comment:
https://www.sqlite.org/c3ref/initialize.html

> Workstation applications using SQLite normally do not need to invoke either of these routines.

Why do we need to to handle for SQLITE_OMIT_AUTOINIT here? Please leave a comment directly in the code including which platforms are affected.

Please also handle the return value of this function and log any errors that may occur.

Reply:
Mmm....

>  For maximum portability,
** it is recommended that applications always invoke sqlite3_initialize()
** directly prior to using any other SQLite interface.  Future releases
** of SQLite may require this.  In other words, the behavior exhibited
** when SQLite is compiled with [SQLITE_OMIT_AUTOINIT] might become the
** default behavior in some future release of SQLite.

I will add some logging if the return code is not zero